src/main/resources/templates/trips/search_trip.html

```diff
@@ -1,10 +1,33 @@
 <!DOCTYPE HTML>
 <html xmlns:th="https://www.thymeleaf.org">
+
+<head>
+<link rel="stylesheet" href="https://code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
+<script src="https://code.jquery.com/jquery-1.12.4.js"></script>
+<script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
+</head>
+
 <body>
 <p>Search Trips</p>
 <!--/*@thymesVar id="TripSearchRequest" type="edu.drexel.TrainDemo.trips.models.TripSearchRequest"*/-->
+
+<!--<h4 id="selectedDateVal"></h4>-->
+<script>
+$( function() {
+$.datepicker.setDefaults({
+onClose:function(date, inst){
+$("#selectedDateVal").html(date);
+}
+});
+
+$( "#datepicker" ).datepicker();
+});
+</script>
+
 <form action="#" id="userProfileForm" method="post" th:action="@{/trips/search/submit}"
       th:object="${TripSearchRequest}">
+    <label>Departure Date: <input type="text" id="datepicker" th:field="*{date}"></label>
```

Comment:
Does something like th:type="${#date}" work here?
Saw `<p th:text="${#dates.format(standardDate, 'dd-MM-yyyy HH:mm')}"></p>` online at [this](https://www.baeldung.com/dates-in-thymeleaf) link.